class Desc:
    
    manuscript_intro = """

Single-cell resolution genomics data allows us to characterize the repertoire of SARS-CoV-2 infected cell and surrounding cells and offers a high-resolution view into the host immune cell states. Cellular indexing of transcriptomes and epitopes by sequencing (CITE-seq) have coupled the relatively sparse single-cell RNA sequencing (scRNA-seq) signal with robust detection of highly abundant and well-characterized surface proteins using barcoded antibodies. One unique feature of CITE-seq is its capacity to analyze the transcriptomic and proteomic (surface-protein) changes of different cell types. We recently developed SPaRTAN (Single-cell Proteomic and RNA based Transcription factor Activity Network), for exploiting single-cell proteomic (scADT-seq) and corresponding scRNA-seq datasets, both obtained using CITE-seq, to link expression of surface proteins with inferred TF activities. The cell surface phenotype is well-known to immunologists through flow cytometry but signalling downstream of cell surface receptors/co-receptors drives transcriptional and chromatin state changes. It is important to connect the “cell surface phenotype” to downstream transcriptional programs (e.g., transcription factors) and resulting transcriptomic phenotypes for delineating cell states. 

Many COVID-19 related CITE-seq data have been generated for public research. Most CITE-seq datasets are generated using peripheral blood mononuclear cells (PBMCs) from subjects. Connecting upstream signal transduction (e.g., cell surface receptors) to downstream transcription factors (TFs) at a single-cell level would help decode cell states and characterize SARS-CoV-2 infected cell and surrounding cells and how they contribute to developing COVID-19 severe disease. We applied SPaRTAN to COVID-19 CITE-seq datasets to predict the coupling of surface receptors with TFs across host immune cells. As a resource to enable the research community to investigate different aspects of our analysis, we create a webpage for each study supplying surface protein expression and SPaRTAN-predicted TF activities of each cell studied for each dataset. COVID-19db of Immune Cell States is a unique and valuable resource for testing hypotheses and identifying proteins or transcription factors that could be targeted by new treatments of COVID-19 patients. 

"""
    
    
    litcovid = "LitCovid is a literature hub for tracking up-to-date scientific information about the 2019 novel Coronavirus. It is the most comprehensive resource on the subject, providing a central access to 298,429 (and growing) relevant articles in PubMed. The articles are updated daily and are further categorized by different research topics (e.g. transmission) and geographic locations. https://www.ncbi.nlm.nih.gov/research/coronavirus/"
    
    tf =  "Search activity of a TF across COVID-19 patients and health individuals, depending on availability. For a single TF, COVID-19db supports exploring its activity for each cell type, correlated surface proteins, and relevant literature."
    
    ds = "For a single CITE-seq dataset, COVID-19db provides meta information, cell-type annotation, transcription factor activity and surface protein visualization, correlation between transcription factors and surface proteins by each patients, relevant literature, protein-protein interaction results."
    
    # home = "A comprehensive web resource linking surface proteins to transcription factors to delineate cellular heterogeneity of host response during SARS-CoV-2 infection"
    
    # home = "a toolset linking cell surface proteins with transcription factors"
    
    # home = "linkage maps of cell surface proteins and transcription factors in immune cells"
    
    home = """
        The highly contagious SARS-CoV-2 and its associated disease (COVID-19) are a threat to 
        global public health and economies. To develop effective treatments for COVID-19, we must 
        understand the host cell types, cell states and regulators associated with infection and 
        pathogenesis such as dysregulated transcription factors (TFs) and surface proteins, 
        including signaling receptors. 
        To link cell surface proteins with TFs, we recently developed 
        SPaRTAN (Single-cell Proteomic and RNA-based Transcription factor Activity Network) by 
        integrating parallel single-cell proteomic and transcriptomic data based on Cellular Indexing 
        of Transcriptomes and Epitopes by sequencing (CITE-seq) and gene cis-regulatory 
        information. We apply SPaRTAN to CITE-seq datasets from patients with varying degrees of
        COVID-19 severity and healthy controls to identify the associations between surface proteins
        and TFs in host immune cells. Here, we present a web server containing cell surface protein expression, 
        SPaRTAN-inferred TF activities, and their associations with major host immune cell types. 
        The data include four high-quality COVID-19 CITE-seq datasets with a toolset for user-
        friendly data analysis and visualization. We provide interactive surface protein and TF 
        visualizations across major immune cell types for each dataset, allowing comparison 
        between various patient severity groups for the discovery of potential therapeutic targets and
        diagnostic biomarkers.
    """
    
    intro_1 = """COVID-19 requires a permissive cellular environment and host cellular factors for induction 
        of viral transcription. A variety of factors provide a favourable environment for COVID-19 
        infection. The most notable are the set of transcription factors (TFs) as COVID-19 does not 
        have its own transcription machinery and is fully dependent on host TFs to complete its viral 
        life cycle. Further, COVID-19 infection of host cells causes dramatic changes in the immune 
        response. A better understanding of altered cell-type specific TFs downstream of signalling 
        pathways may allow us to develop accurate prognostic markers that can predict the effect of 
        COVID-19 infection on immune responses and identify novel targets for developing more 
        effective therapies for COVID-19. Single-cell muti-omics offers a high-resolution view into the cell and cellular environment. 
        Recent breakthroughs have coupled the relatively sparse scRNA-seq signal with robust 
        detection of highly abundant and well-characterized surface proteins using index sorting 
        and barcoded antibodies such as cellular indexing of transcriptomes and epitopes by 
        sequencing (CITE-seq) thereby permitting better cell type discrimination. """
    
    intro_2 = """Many COVID-19 related CITE-seq data have been generated for public research. Most 
        CITE-seq datasets are generated using peripheral blood mononuclear cells (PBMCs) from 
        subjects. We applied SPaRTAN to blood (PBMC) COVID-19 CITE-seq datasets to predict 
        the coupling of signaling receptors with context-specific TFs. Application of SPaRTAN to 
        healthy individuals and patients with varying severity of COVID-19 help to decipher critical 
        TFs downstream of surface receptors driving different cellular identities for each PBMC cell 
        type and determine commonalities as well as differences of cell context-specific TF and 
        surface receptor across healthy individuals and those manifesting COVID-19. We identify 
        regulators that could assist as predictors for patient survivability of serious COVID-19 
        infection."""
        
        
    analysis = """The distributions of the surface protein expression and TF activities were organized by disease subgroups. Specifically, we used ridge density plots for each dataset and the R package ggridges (version: 0.5.3). The significance of the difference between the disease subgroups in each cell type is evaluated through the Mann-Whitney U test. To explore the associations between inferred TF activities and surface protein expression at a single-cell level, we computed Spearman correlation coefficients (rho) between (inferred) TF activity and surface protein expression for each TF-surface protein pair within each cell-type per sample. We used heatmap to display correlations. For clarity, pairwise Spearman correlation values below 0.3 in all samples are filtered. Similarly, to explore the associations between inferred TF activities and target gene expression, we computed Spearman correlation coefficients (rho) between (inferred) TF activity and gene expression for each TF-gene pair within each cell-type per sample. For clarity, pairwise Spearman correlation values below 0.3 in all samples are filtered.
    """
    
    spartan = """SPaRTAN (Single-cell Proteomic and RNA based Transcription factor Activity Network) provides a mechanistically inspired approach for integrating cell-specific transcriptomic and proteomic data with regulatory genomics resources, representing a significant advance in the modeling of cell-specific signaling and gene regulatory programs. The cell surface phenotype is well-known to immunologists through flow cytometry but signaling downstream of cell surface receptors/co-receptors drives transcriptional and chromatin state changes. It is important to connect the “cell surface phenotype” to downstream transcriptional programs and resulting transcriptomic phenotypes. SPaRTAN models this flow of information with single cell resolution. The description of the method and some usage examples are available in https://doi.org/10.1101/2020.12.22.423961. SPaRTAN greatly enhances the utility of CITE-seq datasets to uncover TF and cell-surface receptor relationships in diverse cellular states."""
    
    doc_spartan = """SPaRTAN framework uses the expression of surface proteins as a proxy for their activities. Signaling via these proteins converges on particular TFs, whose activities, in turn, regulate the expression of their target genes. Formally, we use a regularized bilinear regression algorithm called affinity regression (AR), which provides a statistical framework when the observed data can be explained as interactions between two kinds of inputs. Here, AR establishes an interaction matrix between surface proteins and TFs that predicts target gene expression. We use curated TF-target gene interactions to determine the set of TFs that potentially regulates each gene. The model captures statistical relationships between surface proteins, TFs, and gene expression. We use the trained interaction matrix to evaluate different views of a CITE-seq dataset, e.g., to predict TF activity from a cell's surface protein expression profile or to predict surface protein expression from a cell's gene expression profile. Intuitively, information flows down from observed surface protein levels through the learned interaction matrix to infer TF activities and observed mRNA expression levels or propagates up through the TF-target gene edges and interaction network to infer surface protein expression."""
    
    spartan_model = """Briefly, our model views the expression of surface proteins as a proxy of their activities; signaling emanating from these proteins converges on particular TFs, whose activities, in turn, regulate the expression of their target genes. Specifically, we use a regularized bilinear regression algorithm called affinity regression (AR), a general statistical framework for any problem where the observed data can be explained as interactions (W) between two kinds of inputs, to establish an interaction matrix between surface proteins (P) and TFs (D) that predicts target gene expression (Y).\n\nSince the model captures statistical relationships between surface proteins, TFs, and gene expression. We can use the trained interaction matrix to obtain different views of a CITE-seq data set; e.g., to predict TF activity from a cell's surface protein expression profile or to predict surface protein expression from a cell’s gene expression profile. Intuitively, information flows down from observed surface protein levels through the learned interaction matrix to infer TF activities and observed mRNA expression levels or propagates up through the TF target-gene edges and interaction network to infer surface protein expression."""
    
    model_overview = """The SPaRTAN framework uses the expression of surface proteins as a proxy for their activities. Signaling via these proteins converges on particular TFs, whose activities, in turn, regulate the expression of their target genes.\n\nFormally, we use a regularized bilinear regression algorithm called affinity regression (AR), which provides a statistical framework when the observed data can be explained as interactions between two kinds of inputs. Here, AR establishes an interaction matrix between surface proteins (P) and TFs that predicts target gene expression (D). We use curated TF-target gene interactions to determine the set of TFs that potentially regulates each gene. The model captures statistical relationships between surface proteins, TFs, and gene expression. We use the trained interaction matrix (W) to evaluate different views of a CITE-seq dataset, e.g., to predict TF activity from a cell’s surface protein expression profile or to predict surface protein expression from a cell’s gene expression profile.\n\nIntuitively, information flows down from observed surface protein levels through the learned interaction matrix to infer TF activities and observed mRNA expression levels or propagates up through the TF-target gene edges and interaction network to infer surface protein expression."""
    
DatasetDesc = {
    'E-MTAB-9357' : 'Abstract: Host immune responses play central roles in controlling SARS-CoV2 infection, yet remain incompletely characterized and understood. Here, we present a comprehensive immune response map spanning 454 proteins and 847 metabolites in plasma integrated with single-cell multi-omic assays of PBMCs in which whole transcriptome, 192 surface proteins, and T and B cell receptor sequence were co-analyzed within the context of clinical measures from 50 COVID19 patient samples. Our study reveals novel cellular subpopulations, such as proliferative exhausted CD8+ and CD4+ T cells, and cytotoxic CD4+ T cells, that may be features of severe COVID-19 infection. \n\nWe condensed over 1 million immune features into a single immune response axis that independently aligns with many clinical features and is also strongly associated with disease severity. Our study represents an important resource towards understanding the heterogeneous immune responses of COVID-19 patients and may provide key information for informing therapeutic development.',
    
    'GSE155673' : 'Abstract: Coronavirus disease 2019 (COVID-19) represents a global crisis, yet major knowledge gaps remain about human immunity to severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2). We analyzed immune responses in 76 COVID-19 patients and 69 healthy individuals from Hong Kong and Atlanta, Georgia, United States. \n\nIn the peripheral blood mononuclear cells (PBMCs) of COVID-19 patients, we observed reduced expression of human leukocyte antigen class DR (HLA-DR) and proinflammatory cytokines by myeloid cells as well as impaired mammalian target of rapamycin (mTOR) signaling and interferon-α (IFN-α) production by plasmacytoid dendritic cells. \n\nBy contrast, we detected enhanced plasma levels of inflammatory mediators—including EN-RAGE, TNFSF14, and oncostatin M—which correlated with disease severity and increased bacterial products in plasma. Single-cell transcriptomics revealed a lack of type I IFNs, reduced HLA-DR in the myeloid cells of patients with severe COVID-19, and transient expression of IFN-stimulated genes. This was consistent with bulk PBMC transcriptomics and transient, low IFN-α levels in plasma during infection. These results reveal mechanisms and potential therapeutic targets for COVID-19.',
    
    'GSE155224' : 'A dysregulated immune response against the SARS-CoV-2 virus plays a critical role in severe COVID-19. However, the molecular and cellular mechanisms by which the virus causes lethal immunopathology are poorly understood. Here, we utilize multi-omics single-cell analysis to probe dynamic immune responses in patients with stable or progressive manifestations of COVID-19, and assess the effects of tocilizumab, an anti-IL-6 receptor monoclonal antibody. Coordinated profiling of gene expression and cell lineage protein markers reveals a prominent type-1 interferon response across all immune cells, especially in progressive patients. An anti-inflammatory signature in monocytes and a pre-exhaustion phenotype in activated T cells are hallmarks of progressive disease. Single-cell T and B cell receptor repertoire analysis reveal a skewed clonal distribution of CD8 T cells and a primary B cell response with possible contribution from memory B cells. Our in-depth immune profiling reveals dyssynchrony of the innate and adaptive immune interaction in progressive COVID-19, which may contribute to delayed virus clearance and has implications for therapeutic intervention.',
    
    'GSE161918' : 'Abstract: Hyperinflammation contributes to lung injury and subsequent acute respiratory distress syndrome (ARDS) with high mortality in severe COVID-19. To understand the underlying mechanisms, we investigated the role of the lung-specific immune response. Here we profiled lymphocytes and myeloid cells in the bronchoalveolar lavage (BAL) fluid and blood of COVID-19 patients. \n\nBy tracking T cell clones across tissues, we identified clonally expanded tissue-resident memory-like Th17 cells (Trm17 cells) in the lung even after viral clearance. These Trm17 cells are characterized by a potentially pathogenic cytokine profile with expression of IL17A and CSF2 (GM-CSF). Interactome analysis revealed that Trm17 cells interact with macrophages and cytotoxic CD8+ T cells, which have been associated with disease severity and lung damage. High IL-17A and GM-CSF protein levels in the serum of COVID-19 patients correlated with severe clinical course. \n\nThis study suggests pulmonary Trm17 cells as one of the orchestrators of the hyperinflammation in severe COVID-19 and that these cells and their cytokines, such as GM-CSF, are promising biomarkers and potential targets for a COVID-19 therapy.',
    
    'E-MTAB-10026' : """The COVID-19 pandemic, caused by SARS coronavirus 2 (SARS-CoV-2), has resulted in excess morbidity and mortality as well as economic decline. To characterise the systemic host immune response to SARS-CoV-2, we performed single-cell RNA-sequencing coupled with analysis of cell surface proteins, providing molecular profiling of over 800,000 peripheral blood mononuclear cells from a cohort of 130 patients with COVID-19. Our cohort, from three UK centres, spans the spectrum of clinical presentations and disease severities ranging from asymptomatic to critical. Three control groups were included: healthy volunteers, patients suffering from a non-COVID-19 severe respiratory illness and healthy individuals administered with intravenous lipopolysaccharide to model an acute inflammatory response. Full single cell transcriptomes coupled with quantification of 188 cell surface proteins, and T and B lymphocyte antigen receptor repertoires have provided several insights into COVID-19: 1. a new non-classical monocyte state that sequesters platelets and replenishes the alveolar macrophage pool; 2. platelet activation accompanied by early priming towards megakaryopoiesis in immature haematopoietic stem/progenitor cells and expansion of megakaryocyte-primed progenitors; 3. increased clonally expanded CD8 + effector:effector memory T cells, and proliferating CD4 + and CD8 + T cells in patients with more severe disease; and 4. relative increase of IgA plasmablasts in asymptomatic stages that switches to expansion of IgG plasmablasts and plasma cells, accompanied with higher incidence of BCR sharing, as disease severity increases. All data and analysis results are available for interrogation and data mining through an intuitive web portal. Together, these data detail the cellular processes present in peripheral blood during an acute immune response to COVID-19, and serve as a template for multi-omic single cell data integration across multiple centers to rapidly build powerful resources to help combat diseases such as COVID-19."""
}   
    